src/writer/journal.py

```
@@ -136,9 +148,52 @@ def _sanitize_data(self, data):
+    title: str,
+    graph: "Graph"
+):
+    parent_record = _parent_journal_record.get()
+
+    try:
+        current_record = JournalRecord(execution_environment, title, graph)
+        _current_journal_record.set(current_record)
+        if parent_record is None:
+            _parent_journal_record.set(current_record)
+        yield current_record
+    except:
```

Ruff (0.14.6)

Do not use bare `except`

(E722)